docs/dnsupdate.rst

```diff
@@ -69,6 +69,15 @@ authorization methods, and you are expected to take care of everything
 yourself. See :ref:`dnsupdate-update-policy` for details and
 examples.
 
+.. _dnsupdate-lua-dnsupdate-policy-script-strict-mode:
+
+``lua-dnsupdate-policy-script-strict-mode``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When turning this setting on, PowerDNS will handle DNS updates as atomic,
+rolling back an update when a single record in an update causes the
+configured lua script to return false, and returning `REFUSED` to the client.
+
```

Comment:
I think it might make more sense to move this block down, to just below `.. _dnsupdate-metadata:`

Reply:
My rst-knowledge is limited, but wouldn't the location then imply that this setting is related to per-zone metadata settings?
(from what i figured, it would be show up when browsing to https://docs.powerdns.com/authoritative/dnsupdate.html#dnsupdate-metadata as the first thing in the viewport)